src/Components/RecentJobs/RecentBox/SingleJob/index.jsx

```diff
@@ -1,30 +1,35 @@
 import { useEffect, useState } from "react"
 import Skill from "./Skill/Skill"
+import { NavLink } from "react-router-dom"
 
-function SingleJob({logo, JobTitle, company, type, salary, skills}) {
+function SingleJob({logo, JobTitle, company, type, salary, skills, id}) {
 
     return (
-        <tr>
-            <td>
-                <div class='flex gap-x-2'>
-                    <div class='object-cover h-20 w-20'>
-                        <img src={logo} alt-text="logo of company" />
+            
+            <tr>
+                <td>
+                <NavLink to={"/JobListing/" + id}>
```

Comment:
You should use the Link component here rather than NavLink - just because you don't need the active class that a NavLink provides.